controllers/config.go

```diff
@@ -0,0 +1,9 @@
+package controllers
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/logger"
+)
+
+var (
+	log = logger.NewDTLogger().WithName("controllers")
```

Comment:
does a logger at this level make sense?
wouldn't it be better to move `dynakube_state.go` into `dynakube/status` so there are no more files on this level?